lib/lita/version.rb

```diff
@@ -1,4 +1,4 @@
 module Lita
   # The current version of Lita.
-  VERSION = "4.6.0"
+  VERSION = "4.6.2"
```

Comment:
Okay, so how can `lita-slack` specifically target the version with this change? Or would it be a case of then merging this change into `4-stable` and bumping the version number there?